mindera-people/Sources/Shared/DTO/Event.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import MinderaPeopleKMMLib
+
+public struct Event: Equatable, Hashable, Identifiable {
+    
+    public var id: UUID
+    public var policy: Policy
+    public var startDate: Date
+    public var endDate: Date
+    public var partOfDay: PartOfDay
+    public var additionalInfo: String?
+    public var includesBreakfast: Bool?
+    public var city: String?
+    public var project: Project?
+    
+    public init(
+        id: UUID,
+        policy: Policy,
+        startDate: Date,
+        endDate: Date,
+        partOfDay: PartOfDay,
+        additionalInfo: String? = nil,
+        includesBreakfast: Bool? = nil,
+        city: String? = nil,
+        project: Project? = nil
+    ) {
+        self.id = id
+        self.policy = policy
+        self.startDate = startDate
+        self.endDate = endDate
+        self.partOfDay = partOfDay
+        self.additionalInfo = additionalInfo
+        self.includesBreakfast = includesBreakfast
+        self.city = city
+        self.project = project
+    }
+}
+
+extension Event: KMMConvertible {
+    
+    public typealias KMMDTO = EventDTO
+    
+    public func toKMM() -> EventDTO {
+        let outputFormat = Date.FormatStyle()
+                .locale(Locale(identifier: "en_US"))
+                .year(.defaultDigits)
+                .month(.twoDigits)
+                .day(.twoDigits)
+                
+        var project: ProjectDTO? = nil
+        
+        if let optionalProject = self.project {
+            project = optionalProject.toKMM()
+        }
+        
+        return EventDTO(
+            id: id.uuidString,
+            policy: policy.toKMM(),
+            startDate: outputFormat.format(self.startDate),
+            endDate: outputFormat.format(self.endDate),
+            partOfDay: partOfDay.toKMM(),
+            additionalInfo: additionalInfo,
+            includesBreakfast: includesBreakfast?.asKotlinBoolean(),
+            city: city,
+            project: project
```

Comment:
```suggestion
            project = project?.toKMM()
```

EventDTO.project is an optional so you can put directly the method instead trying to unwrap 
You can remove :

> var project: ProjectDTO? = nil
> 
>         if let optionalProject = self.project {
>             project = optionalProject.toKMM()
>         }